### DISCUSSION TITLE
在使用 VLESS-XHTTP-REALITY 流式上行时，scMinPostsIntervalMs和scMaxEachPostBytes参数是不是没有作用？

### DISCUSSION THREAD

**Discussion by johnny-jyj** [XTLS/Xray-core#4078] [Q&A]
使用 VLESS-XHTTP-REALITY 流式上行时使用一个长连接进行上传，这是否意味着scMinPostsIntervalMs和scMaxEachPostBytes这两个参数在这时不影响实际的传输行为？

另外XHTTP文档对于"packet-up"的描述中 ”将数据包包装为单个 HTTP POST 请求“，是否应该是基于参数包装为若干HTTP POST 请求？

**Comment by RPRX** [comment]
对，scMaxConcurrentPosts 参数也仅针对 packet-up 模式
